tasks/xpack/security/elasticsearch-security-file.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: set fact manage_file_users
-  set_fact: manage_file_users=es_users is defined and es_users.file is defined and es_users.file.keys() | length > 0
```

Comment:
Why are some of these now `.keys() | list` while others are just `list` with the `.keys()` also removed? I can't figure out why some still need `.keys()` and some don't.

Reply:
just a mistake => fixed

Reply:
There were a few more were it was removed. Which is why I thought it was on purpose. At least 2 more in this PR where it was deleted.

Reply:
Are you sure? I don't see no more `| list` without `.keys()`